test/integration/lerna-publish.test.js

```diff
@@ -11,6 +11,23 @@ const lastCommitMessage = (cwd) =>
   execa.stdout("git", ["log", "-1", "--format=%B"], { cwd }).then(normalizeNewline);
 
 describe("lerna publish", () => {
+  test.concurrent("exists with a exit code 0 when there are no updates", async () => {
```

Comment:
exists => exits.

Shouldn't there be an integration test showing the non-zero exit status results when the update command detects no changes?